### PR TITLE
feat(intercom): Add button to KB article edit page

### PR DESCRIPTION
**Pull request by lise-toggl**
## :star2: What does this PR do?

Adds a button to intercom's article edit page:
image

## :bug: Recommendations for testing

Go to intercom's articles section, open an article and look for the button.

## :memo: Links to relevant issues or information

Requested internally

### CODE REVIEW

File: src/scripts/content/intercom.js [hunk 1]
```diff
@@ -31,3 +31,21 @@ togglbutton.render('.conversation__card__content-expanded__controls .inbox__conv
...
+      const description = elem.textContent;
+      return description ? description.trim() : '';
+    };
+
+    const link = togglbutton.createTimerLink({
+      className: 'intercom',
+      description: descriptionSelector,
+      buttonType: 'minimal'
+    });
+    link.style.margin = '3px 15px';
```
Comment (rylek90) [inline src/scripts/content/intercom.js]: Can't we include it in css instead? 🙏

Comment (shantanuraj) [reply src/scripts/content/intercom.js]: I would refrain from adding more new styles to that common CSS file,
we should ideally allow CSS files for the content scripts and have them live next to the script itself,
having styles in content scripts seems like an acceptable alternative until we move towards independent CSS files.

The monolith CSS file is difficult to reason about and is easy to miss, when refactoring/adding/removing integrations

Comment (lise-toggl) [reply src/scripts/content/intercom.js]: @rylek90 same as :point_up:, I didn't want to add another single use class in there. Are you ok with keeping it in the script?

Comment (rylek90) [reply src/scripts/content/intercom.js]: > we should ideally allow CSS files for the content scripts and have them live next to the script itself

This sounds great 🙌 

> Are you ok with keeping it in the script?

Yeah, sure 😅

**Pull request merged**